api/src/main/resources/workbench-api.yaml

```diff
@@ -5184,9 +5186,12 @@ definitions:
         format: int64
       username:
         type: string
+      accessTierShortNames:
+        type: string
+        description: Short names of access tiers available to the user, comma-delimited if multiple.
```

Comment:
I understand why the reporting model may be using a comma-delimited string, but I think our own models (Profile, adminProfile) should returned structured data. If the client wants to render this as a comma-delimited string, that's a display concern which should be easy to implement client-side.

Reply:
agreed, will update